dune_client/api/extensions.py

```diff
@@ -36,7 +37,7 @@
 POLL_FREQUENCY_SECONDS = 1
 
 
-class ExtendedAPI(ExecutionAPI, QueryAPI):
```

Comment:
I know that we have the `upload_csv` in the "`ExtendedAPI`", but I don't believe the endpoints fall under "Higher level helper methods for faster and easier development on top of the base ExecutionAPI".

Actually, never mind, I think we've done the right thing. The `ExtendedAPI` is a superset of all of the other APIs. If we need more sophisticated functionality we can add functions to `ExtendedAPI`.

Reply:
Yes, that is correct.